cogs5e/models/dicecloud/integration.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+import MeteorClient
+
+from cogs5e.models.dicecloud.client import DicecloudClient
+from cogs5e.models.sheet.integrations import LiveIntegration
+
+log = logging.getLogger(__name__)
+
+CLASS_RESOURCES = ("expertiseDice", "ki", "rages", "sorceryPoints", "superiorityDice")
+
+
+def update_callback(error, data):
+    if error:
+        log.warning(error)
+    else:
+        log.debug(data)
+
+
+class DicecloudIntegration(LiveIntegration):
```

Comment:
Added the LiveIntegration base class to create a framework for future live sheet integrations, instead of having explicit Dicecloud integrations